DebugNativeAppEngineStandardProjectTest#testDebugLaunch is a bit flakey

Test failure seen on Neon:
```
com.google.cloud.tools.eclipse.integration.appengine.DebugNativeAppEngineStandardProjectTest
testDebugLaunch(com.google.cloud.tools.eclipse.integration.appengine.DebugNativeAppEngineStandardProjectTest)  Time elapsed: 9.039 sec  <<< FAILURE!
java.lang.AssertionError: App Engine console not active
Expected: a string containing "App Engine Standard at localhost"
     but: was "No consoles to display at this time."
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at com.google.cloud.tools.eclipse.integration.appengine.DebugNativeAppEngineStandardProjectTest.testDebugLaunch(DebugNativeAppEngineStandardProjectTest.java:123)
```

The relevant code section is:
```java
    SWTBotView consoleView = bot.viewById("org.eclipse.ui.console.ConsoleView"); // IConsoleConstants.ID_CONSOLE_VIEW
    consoleView.show();
    assertThat("App Engine console not active",
        consoleView.getViewReference().getContentDescription(),
        Matchers.containsString("App Engine Standard at localhost"));
```

From the screenshot, it looks like the app hadn't finished starting:
![travis-11443-11443 2-linux-20180119-1557-plugins-com google cloud tools eclipse integration appengine-screenshots-testdebuglaunch com google cloud tools eclipse integration appengine debugnativeappenginestandardprojecttest](https://user-images.githubusercontent.com/202851/35165418-3da7cf5c-fd1c-11e7-8811-54ce0ff10c43.jpeg)

(testing screenshots for the win!)

--- a/plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/DebugNativeAppEngineStandardProjectTest.java
+++ b/plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/DebugNativeAppEngineStandardProjectTest.java
@@ -120,9 +120,8 @@ public class DebugNativeAppEngineStandardProjectTest extends BaseProjectTest {
 
     SWTBotView consoleView = bot.viewById("org.eclipse.ui.console.ConsoleView"); // IConsoleConstants.ID_CONSOLE_VIEW
     consoleView.show();
-    assertThat("App Engine console not active",
-        consoleView.getViewReference().getContentDescription(),
-        Matchers.containsString("App Engine Standard at localhost"));
+    SwtBotTestingUtilities.waitUntilViewContentDescription(
+        bot, consoleView, Matchers.containsString("App Engine Standard at localhost"));
     final SWTBotStyledText consoleContents =
         new SWTBotStyledText(bot.widget(widgetOfType(StyledText.class), consoleView.getWidget()));
     SwtBotTestingUtilities.waitUntilStyledTextContains(bot,

--- a/plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotTestingUtilities.java
+++ b/plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotTestingUtilities.java
@@ -18,12 +18,14 @@ package com.google.cloud.tools.eclipse.swtbot;
 
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.widgets.Event;
+import org.eclipse.swtbot.eclipse.finder.widgets.SWTBotView;
 import org.eclipse.swtbot.swt.finder.SWTBot;
 import org.eclipse.swtbot.swt.finder.waits.DefaultCondition;
 import org.eclipse.swtbot.swt.finder.widgets.SWTBotButton;
 import org.eclipse.swtbot.swt.finder.widgets.SWTBotShell;
 import org.eclipse.swtbot.swt.finder.widgets.SWTBotStyledText;
 import org.eclipse.swtbot.swt.finder.widgets.SWTBotTable;
+import org.hamcrest.Matcher;
 
 /**
  * Provides helper methods to aid in SWTBot testing.
@@ -166,4 +168,21 @@ public class SwtBotTestingUtilities {
     });
   }
 
+  /** Wait until the view's content description matches. */
+  public static void waitUntilViewContentDescription(
+      SWTBot bot, SWTBotView consoleView, Matcher<String> matcher) {
+    bot.waitUntil(
+        new DefaultCondition() {
+
+          @Override
+          public boolean test() throws Exception {
+            return matcher.matches(consoleView.getViewReference().getContentDescription());
+          }
+
+          @Override
+          public String getFailureMessage() {
+            return matcher.toString();
+          }
+        });
+  }
 }

